modules/admin-ui/src/main/webapp/scripts/shared/partials/modals/event-details.html

```diff
@@ -1483,31 +1486,46 @@ <h3>{{ item.name }}</h3>
             <header translate="EVENTS.EVENTS.DETAILS.WORKFLOW_DETAILS.TITLE"><!-- Workflow Details --></header>
             <div class="obj-container">
               <table class="main-tbl">
-                <tr>
-                  <td translate="EVENTS.EVENTS.DETAILS.WORKFLOWS.ID"><!-- ID --></td>
-                  <td>{{ subNavData.id }}</td>
-                </tr>
                 <tr>
                   <td translate="EVENTS.EVENTS.DETAILS.WORKFLOWS.TITLE"><!-- Title --></td>
                   <td>{{ subNavData.title }}</td>
                 </tr>
+                <tr ng-if="subNavData.description">
+                  <td translate="EVENTS.EVENTS.DETAILS.WORKFLOWS.DESCRIPTION"><!-- Description --></td>
+                  <td>{{ subNavData.description }}</td>
+                </tr>
                 <tr>
-                  <td translate="EVENTS.EVENTS.DETAILS.WORKFLOWS.TYPE"><!-- Type --></td>
-                  <td>{{ subNavData.workflow }}</td>
+                  <td translate="EVENTS.EVENTS.DETAILS.WORKFLOWS.SUBMITTED_BY"><!-- Submitted by --></td>
+                  <td>
+                    {{ subNavData.creator.name }}
+                    <span ng-if="subNavData.creator.email">{{'<' + subNavData.creator.email + '>' }}</span>
+                  </td>
                 </tr>
                 <tr>
-                  <td translate="EVENTS.EVENTS.DETAILS.WORKFLOWS.STATE"><!-- State --></td>
-                  <td>{{ subNavData.state | translate }}</td>
+                  <td translate="EVENTS.EVENTS.DETAILS.WORKFLOWS.SUBMITTED_AT"><!-- Submitted at --></td>
+                  <td>{{ subNavData.submittedAt | localizeDate : 'dateTime' : 'medium' }}</td>
                 </tr>
                 <tr>
-                  <td translate="EVENTS.EVENTS.DETAILS.WORKFLOWS.STARTED"><!-- Started --></td>
-                  <td>{{ subNavData.start | localizeDate : 'dateTime' : 'medium' }}</td>
+                  <td translate="EVENTS.EVENTS.DETAILS.WORKFLOWS.STATUS"><!-- Status --></td>
+                  <td>{{ subNavData.status | translate }}</td>
+                </tr>
+                <tr ng-if="subNavData.status !== 'EVENTS.EVENTS.DETAILS.WORKFLOWS.OPERATION_STATUS.RUNNING'">
+                  <td translate="EVENTS.EVENTS.DETAILS.WORKFLOWS.DURATION"><!-- Duration --></td>
+                  <td>{{ subNavData.executionTime | humanDuration }}</td>
+                </tr>
+                <tr with-role="ROLE_ADMIN">
```

Comment:
Why show this only for ROLE_ADMIN?

Reply:
The workflow definition makes only sense if you have access to the workflow definitions in means of configuration of Opencast. To my understanding, only developers and system administrators can change workflow definitions on disk. The assumption here is that those users have unrestricted access to Opencast which implies ROLE_ADMIN.

For all other users, the information seems not relevant.

Reply:
I could imagine that e.g. an org admin would want to check which workflow was used for something and this is the only thing which identified the workflow definition.

Reply:
Ups, workflow id. I meant the wdid below.

For the wiid:
Indeed, this is mostly useful for REST APIs (which do not all require ROLE_ADMIN) in terms of using that to launch something. Well, that or for reporting a problem with a specific workflow ;-)

Reply:
Yep, but normally I would not expect any non-system-administrator to interact with REST APIs. And if there is something that cannot be done in the UI, we should rather implement the missing function in the UI instead of asking whatever users to use REST interfaces to achieve whatever they want.

Problems are - at least in our case - reported on event level, not workflow level: "Processing failed for event X".